assets/src/dashboard/components/pageStructure/navigationComponents.js

```diff
@@ -25,7 +25,7 @@ export const LeftRailContainer = styled.nav`
   justify-content: space-between;
   flex-direction: column;
   z-index: 1;
-  height: inherit;
+  height: calc(100vh - 32px); /* ADMIN_TOOLBAR_HEIGHT = 32 */
```

Comment:
We need to keep track of everywhere we have to use this value. I do like the `ADMIN_TOOLBAR_HEIGHT` comment.